cloudify/tests/test_local_workflows.py

```diff
@@ -1454,7 +1454,7 @@ def test_no_workflow(self):
             self._execute_workflow(workflow_name='does_not_exist')
             self.fail()
         except ValueError as e:
-            self.assertIn("['workflow0']", e.message)
+            self.assertIn("[workflow0]", str(e))
```

Comment:
changing `e.message` to `str(e)` is pretty much unrelated, but hey, less warnings from tests
(e.message is deprecated and throws a warning)